scripts/docker/Dockerfile.release

```diff
@@ -0,0 +1,46 @@
+FROM debian:buster-slim
```

Comment:
it's better to use "Dockerfile" as an extension: `release.Dockerfile`. Other way many linters do not recognize it.